packages/clay-css/src/scss/variables/_utilities.scss

```diff
@@ -753,6 +753,9 @@ $c-prefers-letter-spacing: map-deep-merge(
 	(
 		'.c-prefers-letter-spacing-1': (
 			letter-spacing: 1px !important,
+			'menubar-transparent .nav-link': (
```

Comment:
I think @ethib137's suggestion using the universal selector is better. It's more invasive ensuring we don't get similar issues popping up if someone modifies `letter-spacing`. We still should have `letter-spacing: 1px !important` on `.c-prefers-letter-spacing-1` though.

```
'.c-prefers-letter-spacing-1, .c-prefers-letter-spacing-1 *': (
	letter-spacing: 1px !important,
),
```

should cover all cases. We don't need to worry about specificity with `!important`. Specificity only matters if the original value also has `!important` which we shouldn't use in our component CSS.

It's not related to this issue, but we can't nest `'*'` inside `.c-prefers-letter-spacing-1` due to Sass failing. It's probably an issue with our mixin, but I'm not 100% sure. It requires some research.

Reply:
Thanks for the explanation @pat270. It worked fine within the navbar items but not for the tooltip. 
Is there anything else I can do?

Reply:
@ilzamcmed just change the whole thing to
```
'.c-prefers-letter-spacing-1, .c-prefers-letter-spacing-1 *': (
	letter-spacing: 1px !important,
),
```

Reply:
@pat270  I`ve changed it according to your suggestion and is not working for tooltips.

Reply:
@ilzamcmed I think it's because tooltips are placed outside of the scope for `.c-prefers-letter-spacing-1` in DXP.

Reply:
@ilzamcmed , where is this not working for tooltips? When I test it, it works fine. 

@pat270 the `c-prefers-letter-spacing-1` class is already being added to the body element, so I'm surprised we are seeing any issues.

Reply:
@ethib137, I was testing on the storybook but maybe I was doing something wrong. I've sent Patrick's suggestion.

Reply:
I just tried on the storybook and first time I tried adding it to the wrong body class. You need to make sure it's added the body element inside of the iframe that is rendering the tooltip component. I'm guessing that's the issue you saw.